aws/scripts/py/cache_invalidation.py

```
@@ -0,0 +1,196 @@
+
+def get_cloudfront_client() -> CloudFrontClient:
+    """Get CloudFront client with error handling"""
+    try:
+        return boto3.client("cloudfront")
+    except (NoCredentialsError, PartialCredentialsError) as err:
+        raise RuntimeError("AWS credentials not found or incomplete") from err
+    except Exception as err:
+        raise RuntimeError("Failed to create CloudFront client") from err
+
+
+def _classify_distribution(dist: Dict[str, Any]) -> str | None:
```

Ruff (0.11.9)

Use `dict` instead of `Dict` for type annotation

Replace with `dict`

(UP006)